collector/pkg/component/analyzer/cpuanalyzer/send_trigger.go

```diff
@@ -29,18 +35,18 @@ func ReceiveDataGroupAsSignal(data *model.DataGroup) {
 		})
 		return
 	}
-	if data.Labels.GetBoolValue(constlabels.IsSlow) || data.Labels.GetBoolValue(constlabels.IsError) {
-		duration, ok := data.GetMetric(constvalues.RequestTotalTime)
-		if !ok {
+	if data.Labels.GetBoolValue("isInstallApm") {
+		isInstallApm[uint64(data.Labels.GetIntValue("pid"))] = true
+	} else {
+		if isInstallApm[uint64(data.Labels.GetIntValue("pid"))] && data.Labels.GetBoolValue(constlabels.IsServer) {
 			return
 		}
-		event := SendTriggerEvent{
-			Pid:          uint32(data.Labels.GetIntValue("pid")),
-			StartTime:    data.Timestamp,
-			SpendTime:    uint64(duration.GetInt().Value),
-			OriginalData: data.Clone(),
+	}
+	if data.Labels.GetBoolValue(constlabels.IsSlow) {
+		_, ok := sampleMap.Load(data.Labels.GetStringValue(constlabels.ContentKey) + string(data.Labels.GetIntValue("pid")))
+		if !ok {
+			sampleMap.Store(data.Labels.GetStringValue(constlabels.ContentKey)+string(data.Labels.GetIntValue("pid")), data)
```

Comment:
Run `go test`  and it will report the error: 
> conversion from int64 to string yields a string of one rune, not a string of digits (did you mean fmt.Sprint(x)?)

You should use `strconv.FormatInt()` or `fmtSprint()`.